src/main/resources/site/pages/default/default.html

```diff
@@ -1,11 +1,13 @@
 <html data-th-lang="${language.code}">
 <head>
+  <!-- UA GA tracking script with added GA4 gtag config -->
   <script async="true" data-th-if="${GA_TRACKING_ID}" data-th-src="|https://www.googletagmanager.com/gtag/js?id=${GA_TRACKING_ID}|"></script>
   <script th:inline="javascript" data-th-if="${GA_TRACKING_ID}">
     window.dataLayer = window.dataLayer || [];
     function gtag(){dataLayer.push(arguments);}
     gtag('js', new Date());
     gtag('config', /*[[${GA_TRACKING_ID}]]*/);
+    gtag('config', 'G-RWG24LNZ9T');
```

Comment:
Skal det være samme id for alle miljø?

Reply:
Har bare fått den ene id'en der, den opererer som et tillegg til den eksisterende id'en, så jeg tror ikke man trenger forskjellige til hvert miljø når den universelle tag id'en er forskjellig.